config/locales/es.yml

```diff
@@ -85,6 +85,9 @@ es:
       reject: "rechazar"
       restart: "reanudar"
       clone: "Clonar historia"
+      copy_url: "Copiar URL"
+      copy_id: "Copiar ID"
+      look_history: "Mirar historia"
```

Comment:
I think `view_history` fits better than `look_history` in this case because "look" has the meaning of "observe" instead of "check".

Reply:
I've changed it.